PROJECT_TIERS.md

```diff
@@ -0,0 +1,45 @@
+# Project Tiers
+
+The MapLibre Organization maintains the map rendering frameworks MapLibre GL JS and MapLibre GL Native and fosters an ecosystem of plugins and tools around these libraries to produce the best maps in the world.
+
+The three project tiers Core, Supported, and Hosted describe qualitatively how donated resources are distributed among the projects.
```

Comment:
```suggestion
The three project tiers Core, Supported, and Hosted describe qualitatively how organization resources are distributed among the projects.
```

Reply:
I'd not specify "donated" since the resources may or may not be donated explicitly.